package.json

```diff
@@ -33,6 +33,11 @@
 	],
 	"license": "MIT",
 	"main": "index.js",
+	"files": [
```

Comment:
The files field is dangerous and should NEVER be used; plus, we already use npmignore. If you want files ignored, add them to the publishConfig.ignore list below.

Reply:
> The files field is dangerous and should NEVER be used;

TIL! Any resources to read about this? Couldn't find anything with a quick Google search.